package.json

```diff
@@ -56,10 +67,10 @@
 		"@typescript-eslint/eslint-plugin": "5.60.1",
 		"@typescript-eslint/parser": "5.60.1",
 		"eslint": "^8.43.0",
```

Comment:
Should these updates really be included in this commit?

Reply:
Separated it into another PR: https://github.com/utily/isoly/pull/125